shards/modules/json/json.cpp

```diff
@@ -617,6 +617,14 @@ struct ToJson {
     else
       return Var(_indent);
   }
+  static SHOptionalString help() { return SHCCSTR("This shard takes its input and converts it into a JSON string."); }
+
+  static SHOptionalString inputHelp() {
+    return SHCCSTR("If Pure is set to true, this shard acceps an input of anytype. If Pure is set to false, this shard only "
```

Comment:
This is wrong, pure stands for `pure` json, sort of compatible with JS, you should check the output, non pure is instead shards flavored json, (with shards types info)